### PR TITLE
Add wg installation

### PR DESCRIPTION
**Pull request by piotrmsc**
**Description**

Changes proposed in this pull request:

- Added Readme for wg-installation

**Related issue(s)**

### CODE REVIEW

File: collaboration/wg-installation/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+#  Installation Working Group
+
+The purpose of this working group is to:
+* Publicly represent Kyma installation topics.
+* Facilitate collaboration with external contributors.
+* Discuss and work on installation and upgrade improvements/aspects:
+	* Process 
+		* Design - rethink how it should look like
+		* Complexity - straightforward and declarative apporach with limitted manual steps
+		* Time consumption - focus on how long does it take to install/upgrade kyma
```
Comment (mareknawa) [inline collaboration/wg-installation/README.md:10]: I would expect here also:
* discover and store current baseline for installation time
* monitoring and notifications mechanism about the degradation of installation time

**Comment by mareknawa** [comment]
For the working group, I would expect some concrete exit criteria when we agree that the group achieved its goal and we can close it. E.g.
* installation time not longer than 20min on following environments: <specify environments which the group works on>
* independent components' upgrades
etc.
Work on continuous improvement is more a SIG objective.

File: collaboration/wg-installation/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+* [wg-installation](https://github.com/orgs/kyma-project/teams/wg-installation)
+
+## Owned repositories
+
+* WG does not own (yet) any repository
+
+## Reference
+
+Read the [**Special Interest Group & Working Group**](#special-interest-group-working-group-special-interest-group-working-group) document to learn about the structure of Kyma SIGs and WGs.
```
Comment (mareknawa) [inline collaboration/wg-installation/README.md:49]: fix the link to SIG/WG governance model: [**Special Interest Group & Working Group**](/kyma-project/community/blob/master/collaboration/01-sig-and-wg.md)

File: collaboration/wg-installation/README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+## Members
+
+* [wg-installation](https://github.com/orgs/kyma-project/teams/wg-installation)
+
+## Owned repositories
+
+* WG does not own (yet) any repository
+
+## Reference
```
Comment (mareknawa) [inline collaboration/wg-installation/README.md:47]: I suggest adding as a reference link to GitHub issue with the [proposal of this group](https://github.com/kyma-project/community/issues/464). We have there details about success measures which will be helpful to close the group at some point.

**Comment by kazydek** [comment]
I am updating the doc on SIGs and WGs in [my PR](https://github.com/kyma-project/community/pull/476) so I will add the Installation WG to the list - if you could just add a link to the README, that would be great.

File: collaboration/wg-installation/README.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+* [Piotr Mścichowski](https://github.com/piotrmsc)
+* More leaders will be elected afer the first meeting if proposed
+
+## Members
+
+* [wg-installation](https://github.com/orgs/kyma-project/teams/wg-installation)
+
+# Exit criteria
+
+* Group will work out technical concept and way to install/upgrade kyma meeting all the given requirements mentioned [here](https://github.com/kyma-project/kyma/issues/8664)
```
Comment (bszwarc) [inline collaboration/wg-installation/README.md:45]:
```suggestion
* The group works out the technical concept and the way to install/upgrade Kyma. This concept should meet all the requirements mentioned in the issue describing the [redesign of the kyma-operator](https://github.com/kyma-project/kyma/issues/8664).
```

File: collaboration/wg-installation/README.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+* Members will be inactive or interest will significantly drop down making it very hard to drive and implement topics
```
Comment (bszwarc) [inline collaboration/wg-installation/README.md:46]:
```suggestion
* Members are inactive or interest drops significantly making it very hard to drive and implement topics.
```

File: collaboration/wg-installation/README.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+* All the goals will be met/completed
```
Comment (bszwarc) [inline collaboration/wg-installation/README.md:47]:
```suggestion
* All the goals are achieved/completed.
```

File: collaboration/wg-installation/README.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+* Global product priorities will change
```
Comment (bszwarc) [inline collaboration/wg-installation/README.md:48]:
```suggestion
* There is a change in product priorities.
```

File: collaboration/wg-installation/README.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+## Owned repositories
+
+* WG does not own (yet) any repository
```
Comment (bszwarc) [inline collaboration/wg-installation/README.md:52]:
```suggestion
* The installation working group does not own any repository yet.
```

File: collaboration/wg-installation/README.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+
+## Reference
+
+* Working Group proposal PR : https://github.com/kyma-project/community/issues/464
+
+
```
Comment (bszwarc) [inline collaboration/wg-installation/README.md:58]: remove the black lines :)

File: collaboration/wg-installation/README.md [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+	* Pluggability of components
+	* Error handling and debugging
+	* Underlying technology
+
+
+## Contact
+
+* [Slack](https://kyma-community.slack.com/archives/CD2HJ0E78)
+* [Google group](https://groups.google.com/forum/#!forum/kyma-sig-core)
+* [Open community issues and pull requests](https://github.com/kyma-project/community/labels/area%2Finstallation) - separate label wg-installation to be discussed.
```
Comment (bszwarc) [inline collaboration/wg-installation/README.md:23]:
```suggestion
* [Open community issues and pull requests](https://github.com/kyma-project/community/labels/area%2Finstallation) - a separate **wg-installation** label to be discussed.
```

File: collaboration/wg-installation/README.md [hunk 11]
```diff
@@ -0,0 +1,59 @@
+#  Installation Working Group
+
+The purpose of this working group is to:
+* Publicly represent Kyma installation topics.
+* Facilitate collaboration with external contributors.
+* Discuss and work on installation and upgrade improvements/aspects:
+	* Process 
+		* Design - rethink how it should look like
+		* Complexity - straightforward and declarative apporach with limitted manual steps
+		* Time consumption - focus on how long does it take to install/upgrade kyma
```
Comment (bszwarc) [inline collaboration/wg-installation/README.md:10]:
```suggestion
		* Time consumption - focus on how long it takes to install/upgrade Kyma
```

File: collaboration/wg-installation/README.md [hunk 12]
```diff
@@ -0,0 +1,59 @@
...
+	* Configuration and maintenance 
+	* Observability 
+	* Parallell processing with dependency management
```
Comment (bszwarc) [inline collaboration/wg-installation/README.md:13]:
```suggestion
	* Parallel processing with dependency management	
```

File: collaboration/wg-installation/README.md [hunk 13]
```diff
@@ -0,0 +1,59 @@
...
+
+* [Piotr Bochyński](https://github.com/pbochynski)
+* [Piotr Mścichowski](https://github.com/piotrmsc)
+* More leaders will be elected afer the first meeting if proposed
+
+## Members
+
+* [wg-installation](https://github.com/orgs/kyma-project/teams/wg-installation)
+
+# Exit criteria
```
Comment (bszwarc) [inline collaboration/wg-installation/README.md:43]:
```suggestion
## Exit criteria
```

I would use H2 here as well

File: collaboration/wg-installation/README.md [hunk 14]
```diff
@@ -0,0 +1,59 @@
+#  Installation Working Group
+
+The purpose of this working group is to:
+* Publicly represent Kyma installation topics.
+* Facilitate collaboration with external contributors.
+* Discuss and work on installation and upgrade improvements/aspects:
```
Comment (bszwarc) [inline collaboration/wg-installation/README.md:6]:
```suggestion
* Discuss and work on the installation and upgrade improvements/aspects:
```

**Pull request merged**